tests/test_local_analytics_exporter.py

```diff
@@ -26,14 +26,21 @@ def test_export(self):
         [lp1] = w1.license_pools
         [lp2] = w2.license_pools
         edition1 = w1.presentation_edition
+        edition1.publisher = "A publisher"
+        edition1.imprint = "An imprint"
         edition2 = w2.presentation_edition
         identifier1 = w1.presentation_edition.primary_identifier
         identifier2 = w2.presentation_edition.primary_identifier
-        genres = self._db.query(Genre).all()
+        genres = self._db.query(Genre).order_by(Genre.name).all()
```

Comment:
This made it easier to debug the affinity change, since there was an obvious right order for the genres -- reverse alphabetical order.